jobQueue/setup.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     BULK_CREATE_AUDIT_EVENTS: conf.getBulkCreateAuditEventJobConcurrency,
     BULKUPSERTSAMPLES: conf.bulkUpsertSampleJobConcurrency,
     GET_HIERARCHY: conf.getHierarchyJobConcurrency,
+    BULKDELETESUBJECTS: conf.getBulkDeleteSubjectsJobConcurrency,
```

Comment:
should be BULK_DELETE_SUBJECTS

Reply:
That's funny thought this was weird but just copied the bulk upsert 'convention'... I guess we should change it as well.